### DISCUSSION TITLE
Greenfield: Missing "List Users" API call

### DISCUSSION THREAD

**Discussion by woutersamaey** [btcpayserver/btcpayserver#3166] [Ideas / Feature Requests]
We have
- create user
- delete user
- delete myself
- get my own info

But no "list all users". This would ideally contain the same data as "get my own info" but for everyone. Some suitable permissions should also be chosen.

**Comment by pavlenex** [comment]
https://github.com/btcpayserver/btcpayserver/issues/6727#issuecomment-2897294361
